allensdk/brain_observatory/behavior/swdb/utilities.py

```diff
@@ -76,11 +74,13 @@ def get_mean_df(response_df, conditions=['cell_specimen_id', 'image_name'], flas
         mdf = annotate_mean_df_with_pref_stim(mdf)
 
     # What fraction of individual responses were significant?
-    if flashes:
-        fraction_significant_responses = rdf.groupby(conditions).apply(get_fraction_significant_responses)
-        fraction_significant_responses = fraction_significant_responses.reset_index()
-        mdf['fraction_significant_responses'] = fraction_significant_responses.fraction_significant_responses
+    # if flashes:
```

Comment:
any reason for this to be commented vs removed entirely?

Reply:
fixed in more recent push